src/main/resources/application.yaml

```
@@ -91,3 +101,8 @@ etc-api:
       get-fcst-version:
         second-path: /getFcstVersion
         data-type: JSON
+
+#이메일 인증시 리다이렉트를 위한 사이트 url 설정
```

YAMLlint (1.37.1)

[warning] missing starting space in comment

(comments)